src/PageContent/RequestPickup/PickupSummary.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './PickupSummary.css';
+import truck from '../../icons/green-truck.svg';
+import Map from '../../Map/Map.js';
+
+class PickupSummary extends React.Component {
+    render() {
+        // Render nothing if the "show" prop is false
+        if(!this.props.show) {
+            return null;
+        }
+        return (
+            <div className="backdrop">
+                {/* TODO: fix background opacity. Maybe with iFrame. */}
+                <div className="modal">
+                    <div className="top-line"></div>
+                    <p id="exit" onClick={this.props.onClose}>&times;</p>
+                    <div className="summary-title flex">
+                        <img src={truck} alt="truck"></img>
+                        <p id="title">{this.props.type}</p>
+                    </div>
+                    <div className="wrapper grid">
+                        <div className="details grid">
+                            <p id="subheading">Pickup Details</p>
+                            <p>Start Date: {this.props.startDate}</p>
+                            {/* TODO: determine end date if they chose recurring # times option */}
```

Comment:
actually, we don't always show end date. It depends on what they chose:

- if chose "End by", display something like: "every other Thursday until 10/23/2018"
- if chose "Recur", display "10 pickups requested for every other Thursday" (see Figma)

where,
- "every other" depends on the `repeats` they chose
- "Thursday" depends on the week day of the `startDate`